core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/verticalscale/CoreVerticalScaleActions.java

```diff
@@ -96,7 +96,7 @@ protected void doExecute(ClusterViewContext ctx, CoreVerticalScalingTriggerEvent
                         cloudCredential,
                         cloudStack,
                         cloudResources,
-                        stackVerticalScaleV4Request);
+                        stackVerticalScaleV4Request, stack.getWorkspace().getId());
```

Comment:
please keep the wrapping for consistency

Reply:
done